### ISSUE TITLE
Update Installation Docs For Dockers

### ISSUE THREAD

**Issue by sk9331657** [fossasia/badgeyay#1673]
**I'm submitting a ...**
- [ ] feature request

**Current behaviour:**

**Expected behaviour:**

**Steps to reproduce:**

**Comment by prabhupant** [comment]
i'm working on this one

**Comment by sk9331657** [comment]
@prabhupant  Are u working on it ??

**Comment by abishekvashok** [comment]
What kind of improvement are you guys looking for, it would be awesome if you could explain it.

**Comment by prabhupant** [comment]
@sk9331657 yes i am working on it...i was having some issues with docker deployment. but feel free to work on it if you want :smiley:

**Comment by sk9331657** [comment]
@prabhupant You can continue I am working to improve front end UI.

**Comment by prabhupant** [comment]
@abishekvashok I got some issues while following the installation docs for docker. As i am a beginner myself to docker, i'm trying to read more about it and make the docs more new-comer friendly

**Comment by ParthS007** [comment]
@prabhupant Great you are learning docker concepts. Please take your time  and make PR when you are done with it :+1:
